scripts/installer.sh

```diff
@@ -527,6 +527,14 @@ main() {
 			;;
 		apk)
 			set -x
+			if ! grep -Eq '^http.*\/community$' /etc/apk/repositories; then
```

Comment:
Totally minor: You don't have to `\`-escape the `/`.

Reply:
:+1:  https://github.com/tailscale/tailscale/pull/11950